pass/__init__.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+"""Manage passwords.
+
+This is a 'pass' wrapper extension.
+
+Synopsis:
+    <trigger> generate <location>
+    <trigger> <filter>"""
+
+import fnmatch
+import os
+from albert import *
+
+md_iid = "0.5"
+md_version = "0.5"
+md_name = "Pass"
```

Comment:
please add a md_license and add yourself to md_maintainers (if you want to be a maintainer)

Reply:
Added myself and BSD-3